lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -1521,13 +1521,13 @@ class NGPHistoryParser {
         event.timestamp.offset,
       );
       const bolus = this.cfg.builder.makeSquareBolus()
-        .with_duration(event.deliveredDuration * sundial.MIN_TO_MSEC)
+        .with_duration(event.deliveredDuration * 1000)
         .with_extended(event.deliveredAmount);
 
       if (event.programmedAmount !== event.deliveredAmount ||
         event.programmedDuration !== event.deliveredDuration) {
         bolus.with_expectedExtended(event.programmedAmount)
-          .with_expectedDuration(event.programmedDuration * sundial.MIN_TO_MSEC);
+          .with_expectedDuration(event.programmedDuration * 1000);
```

Comment:
You could use `sundial.SEC_TO_MSEC` instead? I was going to suggest adding a `sundial.MIN_TO_SEC` for the "times 60" use case, but I don't have strong feelings about it.